proxy/ProxyClientSession.h

```diff
@@ -30,6 +30,8 @@
 #include "InkAPIInternal.h"
 #include "http/HttpServerSession.h"
 
+extern volatile bool http_client_session_draining;
```

Comment:
There should be a metric that publishes this state so that operators can tell we are in draining state.